crates/pop-drink/src/lib.rs

```diff
@@ -0,0 +1,13 @@
+pub use drink::*;
```

Comment:
There are quite a few functions in the examples tests which can be moved here to the crate root, so that they can be used across all example contracts and more importantly, all dev contracts.

Reply:
Yes this was definitely the plan. Wanted to see whether we were happy with them first, before providing them to devs.